airbyte-integrations/connectors/destination-bigquery-denormalized/src/main/java/io/airbyte/integrations/destination/bigquery/formatter/GcsBigQueryDenormalizedRecordFormatter.java

```diff
@@ -16,28 +16,22 @@
 public class GcsBigQueryDenormalizedRecordFormatter extends DefaultBigQueryDenormalizedRecordFormatter {
 
   public GcsBigQueryDenormalizedRecordFormatter(
-                                                JsonNode jsonSchema,
-                                                StandardNameTransformer namingResolver) {
+          final JsonNode jsonSchema,
+          final StandardNameTransformer namingResolver) {
     super(jsonSchema, namingResolver);
   }
 
   @Override
-  protected JsonNode formatJsonSchema(JsonNode jsonSchema) {
+  protected JsonNode formatJsonSchema(final JsonNode jsonSchema) {
     var textJson = Jsons.serialize(jsonSchema);
-    /*
-     * BigQuery avro file loader doesn't support DatTime transformation
-     * https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-avro#logical_types Replace
-     * date-time by timestamp
-     */
-    textJson = textJson.replace("\"format\":\"date-time\"", "\"format\":\"timestamp-micros\"");
```

Comment:
This is moved to `DefaultBigQueryDenormalizedRecordFormatter#getBigQuerySchema` because this is not a valid Json schema format.